exercises/bankaccount.js

```diff
@@ -13,6 +20,18 @@ class Account{
 		//returns the new amount in the account
 	}
 	remove( amount ){
+		if (amount > 0){
+			if (amount>this.amount){
+				var amountWithdrawn = this.amount;
+				this.amount-=this.amount;
```

Comment:
you could have set the amountWithdrawn in either case and then changed the amount in one line below